config/config.go

```diff
@@ -206,15 +206,15 @@ type PaymailConfig struct {
 
 // BeefConfig consists of components required to use beef, e.g. Block Headers Service for merkle roots validation
 type BeefConfig struct {
-	// BlockHeaderServiceHeaderValidationURL is the URL for merkle roots validation in Block Headers Service.
-	BlockHeaderServiceHeaderValidationURL string `json:"block_header_service_url" mapstructure:"block_header_service_url"`
-	// BlockHeaderServiceAuthToken is the authentication token for validating merkle roots in Block Headers Service.
-	BlockHeaderServiceAuthToken string `json:"block_header_service_auth_token" mapstructure:"block_header_service_auth_token"`
+	// BlockHeadersServiceHeaderValidationURL is the URL for merkle roots validation in Block Headers Service.
+	BlockHeadersServiceHeaderValidationURL string `json:"block_headers_service_url" mapstructure:"block_headers_service_url"`
```

Comment:
We need to double check if it doesn't affect env variables on test instances, and if it does we need to prepare those changes and apply it.

Reply:
In the mentioned places name of the Block Headers Service is correct, so this PR solves configuration problems there.